src/components/svelte-react.ts

```diff
@@ -67,6 +45,58 @@ export type ReactProps<Props, Events> = Props & {
   [P in IntrinsicProps]?: JSX.IntrinsicElements['div'][P]
 }
 
+const useEventHandlers = (props: any) => {
+  const [el, setEl] = useState<HTMLElement>()
+  const lastValue = useRef<{ [key: string]: (...args: any[]) => any }>({})
+
+  // Handle updating event listeners when props change
+  useEffect(() => {
+    if (!el) return
+    for (const [key, listener] of Object.entries(props) as [
+      string,
+      (...args: any[]) => any
+    ][]) {
+      const match = eventRegex.exec(key)
```

Comment:
Perhaps an optimization so that we're only running the regex once per prop key? Some kind of Map of keyname to boolean representing if it's an event prop, or undefined if we haven't checked the regex yet?

Reply:
Do you mind if we just leave this for now and just see if it's slow?